### PR TITLE
adding code to detect port exhaustion

### PR DESCRIPTION
**Pull request by jebrando**

### CODE REVIEW

File: win32/devdoc/socket_transport_win32_requirements.md [hunk 1]
```diff
@@ -314,13 +314,21 @@ MOCKABLE_FUNCTION(, SOCKET_ACCEPT_RESULT, socket_transport_accept, SOCKET_TRANSP
 
 **SOCKET_TRANSPORT_WIN32_09_070: [** If `sm_exec_begin` does not return `SM_EXEC_GRANTED`, `socket_transport_accept` shall fail and return `SOCKET_ACCEPT_ERROR`. **]**
 
-**SOCKET_TRANSPORT_WIN32_09_071: [** `socket_transport_accept` shall call `select` determine if the socket is ready to be read passing a timeout of 10 milliseconds. **]**
+**SOCKET_TRANSPORT_WIN32_09_071: [** `socket_transport_accept` shall call `select` determine if the socket is ready to be read passing `connection_timeout_ms`. **]**
```
Comment (nishudeshu) [inline win32/devdoc/socket_transport_win32_requirements.md:317]: nit: `to determine`

**Review by nishudeshu — DISMISSED** [review]

File: win32/devdoc/socket_transport_win32_requirements.md (hunk 1 above)
Comment (jebrando) [reply win32/devdoc/socket_transport_win32_requirements.md]: Updated.

**Review by nishudeshu — APPROVED** [review]

**Pull request merged**